.travis.yml

```diff
@@ -1,5 +1,21 @@
-dist: bionic
 sudo: required
+language: minimal
+
+env:
+  global:
+    - DOCKER_CHANNEL="stable"
+    - PLATFORMS="linux/amd64,linux/arm/v7,linux/arm64,linux/s390x,linux/ppc64le"
+    - PREFER_BUILDCTL="1"
+
+before_install:
+  # update docker
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) $DOCKER_CHANNEL"
+  - sudo apt-get update
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
```

Comment:
also need to add docker-ce-cli

Reply:
don't need this atm. it is using buildkit from 19.03 . can fix in follow-up